config/locales/en.yml

```
@@ -328,6 +328,10 @@ en:
             pg_teaching_apprenticeship: Schools
   views:
     all_records: All records
+    apply_invalid_data_view:
+      invalid_answer_summary: "This application contains 1 answer that you need to amend. This is because it was entered in a free text format."
+      invalid_answers_summary: "This application contains %{total_invalid_fields} answers that you need to amend. This is because they were entered in a free text format."
```

Probably a nitpick, but could we be using standard I18n pluralisations here?
```suggestion
      invalid_answers_summary:
        one: "This application contains 1 answer that you need to amend. This is because it was entered in a free text format."
        other: "This application contains %{total_invalid_fields} answers that you need to amend. This is because they were entered in a free text format."
```

---

Defo good idea, done in https://github.com/DFE-Digital/register-trainee-teachers/pull/1150/commits/211708543fdff8bc5b397be33f4bbbbf298ad964